WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -72,78 +72,80 @@
           <TextBlock Text="Merging unmixed coins with mixed ones undoes those mixes." Classes="warningMessage" IsVisible="{Binding LabelExposeCommonOwnershipWarning}" />
         </StackPanel>
       </StackPanel>
-      <controls:ExtendedListBox Items="{Binding Coins}" VirtualizationMode="None" SelectedItem="{Binding Path=SelectedCoin, Mode=TwoWay}">
-        <controls:ExtendedListBox.ContextMenu>
-          <ContextMenu>
-            <!--Enqueuing needs password, TODO: jump to password box OR display pw box in context menu + send button-->
-            <MenuItem Command="{Binding EnqueueCoin}" IsVisible="false">
-              <MenuItem.Header>
-                <Grid>
+      <controls:BusyIndicator IsBusy="{Binding IsCoinListLoading}" Text="Loading...">
```

Comment:
I wonder why we have a large diff here? Seems like the only change was this line

Reply:
The diff is because of the indentation change.

Reply:
@yahiheb makes sense

Reply:
That is correct. Use this in this case
![image](https://user-images.githubusercontent.com/9844978/68878714-2436dd00-0743-11ea-9e97-399af2fbef49.png)

Reply:
This is great.